src/styles/index.scss

```diff
@@ -168,6 +168,14 @@
     line-height: 20px;
   }
 
+  .font-mega-input {
+    font-family: $main-font;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
```

Comment:
we should always be using `rem` instead of `px`. Can you fix this?

Reply:
Hi @samueldominguez, thank you for your feedback. I have made the necessary changes and pushed a new commit.